airbyte-ci/connectors/live-tests/src/live_tests/conftest.py

```diff
@@ -270,6 +272,14 @@ def get_option_or_fail(config: pytest.Config, option: str) -> str:
     pytest.fail(f"Missing required option: {option}")
 
 
+def get_control_version(config: pytest.Config) -> str:
+    if control_version := config.getoption("--control-version"):
```

Comment:
This is partially to do with previous code, but I'm noticing that the pattern is inconsistent between target & control and am wondering if we should clean it up. We give the target version a default (`dev`), but the control version defaults to `None`. I'd propose to default the control to `latest` and handle that here, instead of letting it be `None`. WDYT?

Reply:
I'd prefer to avoid `latest` as much as possible (cf https://github.com/airbytehq/airbyte/pull/41949#discussion_r1679255184)